dataland-frontend/tests/e2e/fixtures/fakeFixtures.ts

```diff
@@ -1,25 +1,43 @@
-const { faker } = require('@faker-js/faker');
+const {faker} = require('@faker-js/faker');
+const backend = require( "../../../build/clients/backend/backendOpenApi.json")
```

Comment:
Ich habe jetzt schon öfters gesehen, dass die spec oder z.B. auch das array mit den Backend-Indizes in dem Index-Mapper einer Variable `backend` zugewiesen werden.
Mich als "Erst-Leser" hat das jetzt immer ziemlich verwirrt. 
Wenn es nicht mit zu großem Aufwand verbunden ist, würde ich vorschlagen, dass wir die Variable sprechender benennen?

Reply:
von meiner Seite resolvet. @arjmandi-hamid  auch zufrieden?